merge/multiple_appliers_test.go

```diff
@@ -554,11 +554,10 @@ func testMultipleAppliersFieldUnsetting(t *testing.T, v1, v2, v3 fieldpath.APIVe
 					`,
 				},
 			},
-			Object: typed.YAMLObject(fmt.Sprintf(`
+			Object: typed.YAMLObject(`
 				struct:
 				  name: a
-				  complexField_%s: null
```

Comment:
is this case still tested?

Reply:
Joe had to write the test with the "dangling" struct only because of this bug that I'm fixing. This is the bug fix :-)